### PR TITLE
Fixes #368 - ca$he bust all compiled assets

### LINKED ISSUE #368: Add cache buster to all production minified assets.

### PR DESCRIPTION
**Pull request by miketaylr**
Let's consistently cache bust all compiled JS + CSS assets. And let's only update the hash if a file actually changed. Before we were doing it based on a deploy timestamp which is not very efficient.

### CODE REVIEW

**Comment by miketaylr** [comment]
Oops. I forgot to r? @karlcow

File: webcompat/helpers.py [hunk 1]
```diff
@@ -27,6 +31,37 @@ def format_delta_filter(timestamp):
...
+    '''Return the md5 checksum for a given file path.'''
+    with open(file_path, 'rb') as fh:
+        m = hashlib.md5()
+        while True:
+            # only read in 8k of the file at a time
+            data = fh.read(8192)
+            if not data:
+                break
+            m.update(data)
+        return m.hexdigest()
```
Comment (karlcow) [inline webcompat/helpers.py]: just for my own understanding and to be sure it was doing the right thing.

``` python
>>> import hashlib
>>> m = hashlib.md5()
>>> m
<md5 HASH object @ 0x107a0c230>
>>> m.hexdigest()
'd41d8cd98f00b204e9800998ecf8427e'
>>> fullstring = 'ABC'
>>> n = hashlib.md5()
>>> n.hexdigest()
'd41d8cd98f00b204e9800998ecf8427e'
>>> n.update(fullstring)
>>> n.hexdigest()
'902fbdd2b1df0c4f70b4a5d23525e932'
>>> m.update('A')
>>> m.update('B')
>>> m.update('C')
>>> m.hexdigest()
'902fbdd2b1df0c4f70b4a5d23525e932'
>>> m.hexdigest() == n.hexdigest()
True
```

Good to go!

**Comment by karlcow** [comment]
r+ @miketaylr Very cool. :sunflower:

**Comment by miketaylr** [comment]
Thanks!

**Pull request merged**